pyproject.toml

```diff
@@ -0,0 +1,2 @@
+[tool.setuptools_scm]
+write_to = "boutdata/_version.py"
```

Comment:
I guess this works even if there is a underscore? It's a bit hard to tell from just reading the GitHub source code 😅

Reply:
Yes, that's intentional. The `_version.py` is dynamically generated when creating the package to install/upload to PyPi, so it's like a 'hidden' file. The underscore means it shouldn't conflict with regular code files that might in principle be created in future. I don't know what would happen if it used `version.py` and someone created a `version.py` file for another reason, but probably nothing good!